nylas/models/drafts.py

```diff
@@ -0,0 +1,162 @@
+        object: The type of object.
+        thread_id (Optional[str]): The thread that this message belongs to.
+        subject (Optional[str]): The subject of the message.
+        to (Optional[List[EmailName]]): The recipients of the message.
+        cc (Optional[List[EmailName]]): The CC recipients of the message.
+        bcc (Optional[List[EmailName]]): The BCC recipients of the message.
+        reply_to (Optional[List[EmailName]]): The reply-to recipients of the message.
+        unread (Optional[bool]): Whether the message is unread.
+        starred (Optional[bool]): Whether the message is starred.
+        snippet (Optional[str]): A snippet of the message body.
+        body (Optional[str]): The body of the message.
+        attachments (Optional[List[Attachment]]): The attachments on the message.
+        folders (Optional[List[str]]): The folders that the message is in.
+        created_at (Optional[int]): Unix timestamp of when the message was created.
+    """
+
+    object: str = "draft"
+
+
+class TrackingOptions(TypedDict):
+    """
+    The different tracking options for when a message is sent.
+
+    Attributes:
+        label: The label to apply to tracked messages.
+        links: Whether to track links.
+        opens: Whether to track opens.
+        thread_replies: Whether to track thread replies.
+    """
+
+    label: NotRequired[str]
+    links: NotRequired[bool]
+    opens: NotRequired[bool]
+    thread_replies: NotRequired[bool]
+
+
+class CreateDraftRequest(TypedDict):
+    """
+    A request to create a draft.
+
+    Attributes:
+        thread_id: The thread that this message belongs to.
+        subject: The subject of the message.
+        to: The recipients of the message.
+        cc: The CC recipients of the message.
+        bcc: The BCC recipients of the message.
+        reply_to: The reply-to recipients of the message.
+        unread: Whether the message is unread.
+        starred: Whether the message is starred.
+        snippet: A snippet of the message body.
+        body: The body of the message.
+        attachments: The attachments on the message.
+        send_at: Unix timestamp to send the message at.
+        reply_to_message_id: The ID of the message that you are replying to.
+        tracking_options: Options for tracking opens, links, and thread replies.
+    """
+
+    body: NotRequired[str]
+    thread_id: NotRequired[str]
+    subject: NotRequired[str]
+    snippet: NotRequired[str]
+    to: NotRequired[List[EmailName]]
+    bcc: NotRequired[List[EmailName]]
+    cc: NotRequired[List[EmailName]]
+    reply_to: NotRequired[List[EmailName]]
+    attachments: NotRequired[List[CreateAttachmentRequest]]
+    unread: NotRequired[bool]
+    starred: NotRequired[bool]
+    send_at: NotRequired[int]
+    reply_to_message_id: NotRequired[str]
+    tracking_options: NotRequired[TrackingOptions]
+
+
+UpdateDraftRequest = CreateDraftRequest
+""" A request to update a draft. """
+
+
```

Comment:
might be worth commenting here about the functional declaration being because of python keywords "from" and "in" for future readers if they don't read the messages models first

Reply:
Good idea, adding it!